docs/docs/get-started/client-setup.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: Client Setup
+nav_order: 2
+parent: Get Started
+---
+
+# Client Setup and Usage
+
+---
+
+## Adding Devices
+Add a device through the Web UI by clicking the "Add Device" button under `/devices` or `/users`. Once the device profile is created, you can send the WireGuard configuration file to the client.
+
+## Client Instructions
+Clients should download the latest version of the WireGuard client from https://www.wireguard.com/install/. Once installed, clients can import the tunnel via the `.conf` file sent by the administrator.
+
+![Client Instructions](https://user-images.githubusercontent.com/52545545/147296703-a7fa8fce-33bb-42ef-83b3-1b8e9047d1d5.gif)
```

Comment:
Fixed image link

Reply:
Is this mainly to keep the size of the repo small? The process to update many images seems slightly more cumbersome than hosting in the repo. We would need to upload each image into GitHub user content and then find the place in the docs where we need to overwrite for each image. If we hosted in the repo we could have a static reference that I can overwrite.

Reply:
For keeping references to image URLs, maybe [jekyll variables](https://www.ayushsharma.in/2021/08/using-variables-in-jekyll-to-define-custom-content) would work well?

Yeah we want to keep the size of the repo small, and we want to keep large (>50kb) binary files out of the repository as much as possible, especially if they're expected to change often.